doc/governance/index.rst

```diff
@@ -0,0 +1,72 @@
+Ceph Governance
+---------------
+
+Key principles
+==============
+
+ * Decision-making is consensus-driven by those who show up.
+ * Leadership roles are defined primarily by responsibility, not prestige or seniority.
+   * It is normal and healthy for these roles to be passed on to others
+ * Everyone's role is ultimately to serve the users and participation
+   is voluntary.
+
+Bodies
+------
+
+Ceph Executive Council
+======================
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+
+ * Arbiter in cases where decisions cannot be reached by consensus
+ * Distribute key responsibilities amongst themselves or others
+ * Point of contact for the project
+ * Representatives for Ceph foundation board meetings
+ * Ensure things get done
+
+Membership
+^^^^^^^^^^
+
+ * 3 people
+ * Elected by steering committee
+   * Candidates self-nominate or are nominated by other members
+   * Discussion of how roles/responsibilities may be delegated
+   * Ranked-choice vote by steering committee
+ * 2 year terms, with one member staggered
+
+Ceph Steering Committee
+=======================
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+
+ * Elect executive council
+ * Amend governance model (by majority vote)
+ * Meet regularly to discuss and decide on tactical and strategic project
+   improvements
+
+Membership
+^^^^^^^^^^
+
+ * Developers, users, community members
+ * Voting members can be nominated and added/removed by existing
+   members (2/3 majority)
```

Comment:
I removed the description of size and term here since I'm not sure they're needed - we've been operating well with a larger clt, even if not everyone can make the meetings (e.g. due to timezone). This lack of restriction is closer to python's 'core team' model.